packages/multi-select-combo-box/src/vaadin-multi-select-combo-box-mixin.js

```diff
@@ -270,6 +270,7 @@ export const MultiSelectComboBoxMixin = (superClass) =>
         placeholder: {
           type: String,
           observer: '_placeholderChanged',
+          reflectToAttribute: true,
```

Comment:
There were some other mixin changes like this, and for the selected attribute in tabs. This is to satisfy the snapshot tests?